src/main/java/com/schedule/workout/workoutScheduler/database/model/WorkoutDB.java

```diff
@@ -0,0 +1,110 @@
+package com.schedule.workout.workoutScheduler.database.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import javax.validation.constraints.*;
+
+@Entity
+@Table(name="workouts")
+public class WorkoutDB {
+
+    @Id
+    @Column(name = "workout_id")
+    private String id;
+
+    @NotEmpty
+    @Size(min=2, max=20)
+    @Column(name = "name")
+    private String name;
+    @NotEmpty
+    @Size(min=2, max=80)
+    @Column(name = "description")
+    private String description;
+    @NotNull
+    @Min(value = 20)
+    @Max(value = 60)
+    @Column(name = "duration")
+    private Integer duration;
+    @NotNull
+    @Min(value = 1)
+    @Max(value = 5)
+    @Column(name = "difficulty")
+    private Integer difficulty;
+
```

Comment:
If you want to create a section as in this case for this @ManyToOne, 1 space is enough